com.github.rajsolai.textsnatcher.yml

```diff
@@ -0,0 +1,104 @@
+app-id: com.github.rajsolai.textsnatcher
+
+runtime: org.gnome.Platform
+runtime-version: "41"
+base: io.elementary.BaseApp
+base-version: "juno-21.08"
+sdk: org.gnome.Sdk
+command: com.github.rajsolai.textsnatcher
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--system-talk-name=org.freedesktop.Accounts"
+
+modules:
+  - name: libportal
+    buildsystem: meson
+    builddir: true
+    config-opts:
+      - "-Dbackends=gtk3"
+      - "-Ddocs=false"
+    sources:
+      - type: git
+        url: https://github.com/flatpak/libportal.git
+        branch: master
```

Comment:
One thing: this should use a specific commit, instead of a branch, to ensure reproducible builds.

Reply:
Sure I will change it thanks @barthalion